includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -843,6 +843,30 @@ public function prepare_source( $user_id, $force_save_source = false, $existing_
 			$source_object = self::get_source_object( wc_clean( wp_unslash( $_POST['stripe_source'] ) ) );
 			$source_id     = $source_object->id;
 			$customer->maybe_create_customer();
+
```

Comment:
I would move this entire new block to a new private method, just to keep it easier to read it in the future 👍

Reply:
Just noting that I'll address this in the next PR. We still need to fix https://github.com/woocommerce/woocommerce-gateway-stripe/issues/3240 and I suspect we'll need to reuse this code and so refactoring it will be necessary.